pages/article/[id].js

```diff
@@ -338,7 +336,7 @@ function ArticlePage() {
             >
               {similarArticles.map(({ node }) => (
                 <div key={node.id} className={classes.similarMessageContainer}>
-                  <article className={classes.text}>{node.text}</article>
+                  <article className={classes.text}>{nl2br(node.text)}</article>
```

Comment:
The desktop now looks good in line clamp :)

![image](https://user-images.githubusercontent.com/108608/82786769-c294bf80-9e97-11ea-8ad1-f492d20bb394.png)

However, given the limited lines to display in this section, I think we don't need to add `nl2br`, so that we can reveal more text to the user.

Reply:
> The desktop now looks good in line clamp :)
> 
> However, given the limited lines to display in this section, I think we don't need to add `nl2br`, so that we can reveal more text to the user.
> 
> ![image](https://user-images.githubusercontent.com/108608/82786769-c294bf80-9e97-11ea-8ad1-f492d20bb394.png)